docs/journal/01-cold-storage.md

```diff
@@ -1,74 +1,90 @@
+![plan for this first milestone](media/01-cold-storage/logical-architecture-for-milestone01.png)
 
-# Fabrikam's Perspective
-Fabrikam's engineering team has very little experience with high-scale event-oriented systems. They want to make sure that they are [asking the right questions][orientation]. They also need to get a minimum viable product deployed.
 
-For their [first milestone][milestone] they want to:
-- Ingest a representative set of simulated events.
-- Store the raw data for all events for later analysis. [:arrow_right:][cold-storage]
-- Increase the scale until they match the targets for their customer [:arrow_right:][increase-scale]
+- [Common requirements](#common-requirements)
+- [Fabrikam requirements](#fabrikam-requirements)
+- [Exploring solutions for ingestion and processing](#exploring-solutions-for-ingestion-and processing)
+- [Exploring solutions for cold storage](#exploring-solutions-for-cold-storage)
+- [Reference implementation](#reference-implementation)
+- [Lessons learned](#lessons-learned)
 
-They need a basic implementation that maps to the highlighted components of the logical architecture:
+## Common requirements
 
-![plan for this first milestone](media/01-cold-storage/logical-architecture-for-milestone01.png)
+Here are some common requirements for ingestion and cold storage that any IoT solution will at least need to consider:
+
+- Achieving scale. That means both hitting the expected target data rate, but also designing the system to be scalable in the future.
+- Authentication. Only authenticated devices can send data. (Provisioning devices is a separate scenario, although closely related.)
+- Protecting the data stream.
+- Storing the raw event data in long-term storage. Cold storage must hold a large volume of historical data indefinitely, at a reasonable cost. 
+
+  
+## Fabrikam requirements
+
+:memo: For more about our Fabrikam scenario, see [Introducing the Journey][00-intro]. 
 
+Here are some specific requirements for our scenario:
 
-## Devices
+- As an initial target, the system must handle up to 100,000 devices, each device sending 1 reading every minute. That's approximately 1,667 events per second. This is a minimum load &mdash; the system should be scalable beyond this target. 
+- Events might be time critical, so the Event Processing component must handle this volume of data without introducing any backlog into the system.
+- The event data stream must be protected. This means not only securing the content of the events, but also masking the *absence* of events. For example, if the motion detectors in a house are not streaming any events, an attacker can conclude that nobody is home.
```

Comment:
I don't think that this is correct place for this requirement. This is more about a malicious agent detecting activity prior to the ingestion process (that is, while events are travelling on a public network to the cloud-hosted gateway).